pyzo/tools/__init__.py

```diff
@@ -210,7 +210,7 @@ def loadToolInfo(self):
                     if i<0: continue
                     line = line.rstrip("\n").rstrip("\r")
                     line = line[i+1:].strip(" ")
-                    toolName = line.strip("'").strip('"')
+                    toolName = eval(line)
```

Comment:
I really don't like the use of eval unless in places where it makes sense :D

Reply:
aah, now I realize that this is what makes the other changes possible. Not a big fan of `eval()` but I guess its use is warranted here ...